src/containers/MyNdla/components/Avatar.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import styled from '@emotion/styled';
+import { fonts, colors } from '@ndla/core';
+import { useTranslation } from 'react-i18next';
+import { useArenaUser } from '../arenaMutations';
+import { FeideUserApiType } from '../../../interfaces';
+
+type AvatarProps = {
+  myProfile?: boolean;
+  user: FeideUserApiType | undefined;
+};
+
+const StyledAvatarContainer = styled.div`
+  width: 48px;
+  height: 48px;
+  border-radius: 48px;
+  border: 1px solid ${colors.brand.tertiary};
+  &[data-myprofile='true'] {
+    width: 250px;
+    height: 250px;
+    border-radius: 246px;
```

Comment:
Gir ikkje dette samme effekten? Poenget er rundt bilde ikkje sant?
```suggestion
    border-radius: 50%;
```